astropy/nddata/nddata.py

```diff
@@ -15,93 +15,95 @@
-        marked by ``False`` and invalid ones with ``True``.
+        Mask for the dataset. Masks should follow the ``numpy`` convention that
+        **valid** data points are marked by ``False`` and **invalid** ones with
+        ``True``.
         Defaults to ``None``.
 
     wcs : any type, optional
-        A world coordinate system (WCS) for the dataset.
+        World coordinate system (WCS) for the dataset.
         Default is ``None``.
 
     meta : `dict`-like object, optional
-        Meta information about the dataset. If no meta is provided an empty
-        `collections.OrderedDict` is created.
+        Additional meta informations about the dataset. If no meta is provided
+        an empty `collections.OrderedDict` is created.
+        Default is ``None``.
 
     unit : `~astropy.units.Unit`-like or str, optional
-        Unit for the dataset.
+        Unit for the dataset. Strings that can be converted to a
+        `~astropy.units.Unit` are allowed.
         Default is ``None``.
 
     copy : `bool`, optional
-        Save the attributes as copy or as reference. ``True`` copies every
-        attribute before saving it while ``False`` tries to save every
-        parameter as reference. Note however that it is not always possible to
-        save each input as reference.
+        Save the attributes as copy? ``True`` copies every attribute before
+        saving it while ``False`` tries to save every parameter as reference.
+        Note however that it is not always possible to save the input as
+        reference.
         Default is ``False``.
 
+        .. versionadded:: 1.2
+
     Raises
     ------
-    TypeError:
-        In case any parameter does not fulfill the classes restrictions.
+    TypeError
+        In case ``data`` or ``meta`` don't meet the restrictions.
+
+    Attributes
+    ----------
+    meta : `dict`-like
+        Additional meta informations about the dataset.
 
     Notes
     -----
-    Each attribute can be accessed through the homonymous instance attribute,
-    such as data in a `NDData` object can be accessed through the ``data``
-    attribute.
-
-    For example::
+    Each attribute can be accessed through the homonymous instance attribute:
+    ``data`` in a `NDData` object can be accessed through the `data`
+    attribute::
 
         >>> from astropy.nddata import NDData
         >>> nd = NDData([1,2,3])
         >>> nd.data
         array([1, 2, 3])
 
-    Given an implicit parameter and an explicit one during initialization, for
-    example the ``data`` is a `~astropy.units.Quantity` and the unit parameter
-    is not None. Then the implicit parameter is replaced (without conversion)
-    by the explicit one and a warning is issued::
+    Given a conflicting implicit and an explicit parameter during
+    initialization, for example the ``data`` is a `~astropy.units.Quantity` and
+    the unit parameter is not None. Then the implicit parameter is replaced
```

Comment:
None should have double backticks around it, also please change the period to a comma and lowercase the "Then"